modules/ugdk-core/src/time/timeaccumulator.cc

```diff
@@ -9,7 +9,7 @@ namespace ugdk {
 namespace time {
 
 TimeAccumulator::TimeAccumulator(int duration_miliseconds) {
-    handler_ = manager();
+    handler_ = &manager();
```

Comment:
For this to work, you need to change `TimeAccumulator::handler_`'s type to `Manager&` and initialize it in the constructor's member initialization list, as in:

```c++
TimeAccumulator::TimeAccumulator(int duration_milliseconds) handler_(manager()) { ... }
```

Actually, this class doesn't even need to keep a reference to the manager. But let's leave this to another issue.